Refit/RestMethodInfo.cs

```diff
@@ -68,7 +68,7 @@ internal class RestMethodInfoInternal
             var hma = methodInfo.GetCustomAttributes(true).OfType<HttpMethodAttribute>().First();
 
             HttpMethod = hma.Method;
-            RelativePath = hma.Path;
+            RelativePath = ReflectionHelpers.GetPathPrefixFor(targetInterface) + hma.Path;
```

Comment:
@thomaslevesque Agreed, on both points. I will take a look into that.

Reply:
https://github.com/reactiveui/refit/blob/main/Refit/RequestBuilderImplementation.cs#L639-L640

It seems like Refit is also just concatenating the basepath and the relative path (unless the basepath is empty). I agree that this could be more foolproof, however, it seems like a separate issue to me.

Reply:
Agreed. Let's leave it like this for now.